src/driver/amdxdna/aie2_pci.c

```diff
@@ -1044,6 +1044,29 @@ static int aie2_get_force_preempt_state(struct amdxdna_client *client,
 	return 0;
 }
 
+static int aie2_get_resource_info(struct amdxdna_client *client,
+				  struct amdxdna_drm_get_info *args)
+{
+	struct amdxdna_drm_get_resource_info res_info = {};
+	struct amdxdna_dev *xdna = client->xdna;
+	struct amdxdna_dev_hdl *ndev = xdna->dev_handle;
+	const struct amdxdna_dev_priv *pdev = ndev->priv;
```

Comment:
pdev is for "PCIe device" in the driver. Use "priv" instead.
```suggestion
	const struct amdxdna_dev_priv *priv = ndev->priv;
```

BTW, the declare is still not a reverse X'Mas tree. Please update it as well.